src/azure-cli/azure/cli/command_modules/appconfig/_constants.py

```diff
@@ -78,3 +78,8 @@ class JsonDiff:
     ADD = "add"
     DELETE = "delete"
     UPDATE = "update"
+
+
+class ImportMode:
+    ALL = "all"
+    IGNORE_MATCH = "ignore_match"
```

Comment:
>ignore_match [](http://example.com/codeflow?start=20&length=12)

Is there a precedence for using underscore naming in input values? If not, I think hyphen naming might be better since all command line arguments are already hyphen separated. 
`--import-mode ignore-match`

Reply:
I am not sure if there is any documented best practice here. For instance, in the network command module, some enums use [pascal case](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/_params.py#:~:text=arg_type%3Dget_enum_type(%5B%22Basic%22%2C%20%22PathBasedRouting%22%5D)), and others do not. e.g., [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/_params.py#:~:text=skus%20%3D%20%5B,%22Standard_Medium%22)